conf/celeryconfig.py

```diff
@@ -5,13 +5,12 @@
 from celery.schedules import crontab
 
 from chronam.settings import *  # NOQA
-from chronam.settings import INSTALLED_APPS
```

Comment:
This was set to avoid the linter errors on line 29 but it shouldn't break anything given the previous line.